Essentials/src/com/earth2me/essentials/Jails.java

```diff
@@ -160,23 +160,23 @@ public int getCount() {
         @EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
         public void onJailBlockBreak(final BlockBreakEvent event) {
             final User user = ess.getUser(event.getPlayer());
-            if (user.isJailed()) {
+            if (user.isJailed() && !event.getPlayer().hasPermission("essentials.jail.allow-break")) {
```

Comment:
Use Essentials' `user.isAuthorized` for permissions lookups - this uses EssentialsX's perms handling code.